web/app/templates/job.hbs

```diff
@@ -112,45 +113,54 @@
               </td>
             </tr>
           {{/each}}
-        </tbody>
-      </table>
-      <div class="severity-None">*These parameter suggestions are based on latest job execution</div>
-      <hr class="bold-horizontal-line">
-      <div class="container">
-        <div class="row">
-          <div class="col-sm-3 heading bold">Auto Tuning</div>
-          <div class="col-sm-3 heading bold">Tuning Algorithm</div>
-          <div class="col-sm-3 heading bold">Number of Iterations</div>
-        </div>
-      </div>
-      <div class="container">
-        <div class="row">
-          <div class="col-sm-3">
-            <label class="switch">
-              {{input type="checkbox" checked=model.tunein.autoApply disabled=true}}
-              <div class="slider round"></div>
-            </label>
-          </div>
-          <div class="col-sm-3">
-            <select id='soflow' disabled>
-              {{#each model.tunein.tuningAlgorithmList as |algorithm|}}
-                <option value={{algorithm.name}} selected={{eq algorithm.name model.tunein.tuningAlgorithm}}>{{algorithm.name}}</option>
-              {{/each}}
-            </select>
-          </div>
-          <div class="col-sm-3">
-            {{input type="text" class="iteration-count" id="form-workflow-id" name="iterationCount" value=model.tunein.iterationCount disabled=true}}
+          </tbody>
+        </table>
+        <div class="severity-None">*These parameter suggestions are based on latest job execution</div>
+        <hr class="bold-horizontal-line">
+        <div class="container">
+          <div class="row">
+            <div class="col-sm-3 heading bold">Auto Tuning</div>
+            <div class="col-sm-3 heading bold">Tuning Algorithm</div>
+            <div class="col-sm-3 heading bold">Number of Iterations</div>
           </div>
+        </div>
+        <div class="container">
```

Comment:
What's the need for using two different container classes div? IMHO, they can be grouped in a single container with two different rows.